test/cookie.spec.ts

```diff
@@ -1,38 +1,35 @@
-const sinon = require('sinon');
-const { expect } = require('chai');
-const { v4: uuidv4 } = require('uuid');
-const jwtDecode = require('jwt-decode');
-const { default: CustomError } = require('../../utils/CustomError');
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import { v4 as uuidv4 } from 'uuid';
+import jwtDecode from 'jwt-decode';
+import CustomError from '../utils/CustomError';
 
-const cookieRewire = require('../../utils/cookie');
-
-const cookie = cookieRewire.default;
+import cookie from '../utils/cookie';
 
 describe('cookie', () => {
   describe('getSessionDuration', () => {
     it('should return a duration with format X hours', () => {
-      const getSessionDuration = cookieRewire.__get__('getSessionDuration');
-
-      getSessionDuration().should.equal('2 hours');
+      cookie.getSessionDuration().should.equal('2 hours');
     });
   });
 
   describe('getJwtTokenForUser', () => {
     it('should return a json web token with only id', () => {
       const psychologist = uuidv4();
-      const token = cookie.getJwtTokenForUser(psychologist);
+      const token = cookie.getJwtTokenForUser(psychologist, 'randomXSRF');
 
-      token.length.should.be.equal(200);
+      token.length.should.be.equal(233);
```

Comment:
:thinking: isn't that weird that it changed?

Reply:
no, now the getJwtTokenForUser force a xsrf token on second parameters, so it's added to the token